src/test/java/com/microsoft/sqlserver/jdbc/NTLMConnectionTest.java

```diff
@@ -165,12 +174,18 @@ public void testNTLMEncryptedConnection() throws SQLException {
     @Test
     public void testNTLMNonDeafultDatabase() throws SQLException {
         if (ntlmPropsDefined) {

```

Comment:
Need not check this now in all tests. If this is 'false' the test class should fail in `@BeforeAll > setup()` method.